.prettierignore

```diff
@@ -0,0 +1,22 @@
+# IDE files
+.idea/
+.vscode/
+.vs/
+.ionide/
+
+# GitHub templates
+.github/ISSUE_TEMPLATE/**/*
+
+# C#-specific files
+languages/csharp/exercises/**/bin/
+languages/csharp/exercises/**/obj/
+
+# F#-specific files
+languages/fsharp/exercises/**/bin/
+languages/fsharp/exercises/**/obj/
+
+# JavaScript-specific files
+languages/javascript/exercises/**/node_modules/
+
+# TypeScript-specific files
```

Comment:
I'm thinking aloud but it may be worthwhile to solicit contributions from track maintainers for other exemptions. `languages/rust/exercises/**/target/` from Rust, etc.

Reply:
Yes. I'll post a message in the #v3 channel once this is merged.